.rosinstall

```diff
@@ -3,10 +3,10 @@
 # IF YOU WANT TO CHANGE THE ROS ENVIRONMENT VARIABLES
 # USE THE rosinstall TOOL INSTEAD.
 # IF YOU CHANGE IT, USE rosinstall FOR THE CHANGES TO TAKE EFFECT
-- git: {local-name: external_pkgs/phidgets_api, uri: 'https://github.com/ros-drivers-gbp/phidgets_drivers-release.git',
-    version: release/kinetic/phidgets_api/0.7.5-0}
-- git: {local-name: external_pkgs/sicktoolbox, uri: 'https://github.com/ros-drivers/sicktoolbox.git'}
-- git: {local-name: external_pkgs/sicktoolbox_wrapper, uri: 'https://github.com/ros-drivers/sicktoolbox_wrapper.git'}
-- git: {local-name: external_pkgs/zed-ros-wrapper, uri: 'https://github.com/UBC-Snowbots/zed-ros-wrapper.git'}
-- git: {local-name: external_pkgs/ros_arduino_bridge, uri: 'https://github.com/UBC-Snowbots/ros_arduino_bridge.git'}
-- git: {local-name: external_pkgs/nmea_navsat_driver, uri: 'https://github.com/UBC-Snowbots/nmea_navsat_driver.git'}
+- setup-file: {local-name: /opt/ros/kinetic/setup.sh}
```

Comment:
What does changing this line do? @Dulluhan?

Reply:
In theory, ROS will not know how to link the setup.sh under `external_pkgs` to the ros stack. But with recent findings, I am not sure anymore. Will need to check

Reply:
Digging into it, it seems that it doesn't matter too much cause [wstool](http://wiki.ros.org/wstool) ignores this anyways per [REP-126](http://www.ros.org/reps/rep-0126.html).
Also, the hottest thing in town is `wstool` since it is the successor to `rosws` which succeeds `rosinstall`.

Reply:
The devs also do admit `rosinstall` was incredibly powerful but had confusing commands and external effects. Its iterations are made to be simpler with commands having clear single responsibilities.